adders/prettier/config/adder.ts

```diff
@@ -0,0 +1,59 @@
+import { dedent, defineAdderConfig } from "@svelte-add/core";
+import { options } from "./options";
+
+export const adder = defineAdderConfig({
+    metadata: {
+        id: "prettier",
+        name: "Prettier",
+        description: "An opinionated code formatter",
+        environments: { svelte: true, kit: true },
+        website: {
+            logo: "./prettier.svg",
+            keywords: ["prettier", "code", "formatter", "formatting"],
+            documentation: "https://prettier.io/docs/en/",
+        },
+    },
+    options,
+    integrationType: "inline",
+    packages: [
+        { name: "prettier", version: "^3.1.1", dev: true },
+        { name: "prettier-plugin-svelte", version: "^3.1.2", dev: true },
+    ],
+    files: [
+        {
+            name: () => `.prettierrc`,
+            contentType: "text",
+            content: () => {
+                return dedent`
+                {
+                    "useTabs": true,
+                    "singleQuote": true,
+                    "trailingComma": "none",
+                    "printWidth": 100,
+                    "plugins": ["prettier-plugin-svelte"],
+                    "overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
+                }`;
+            },
+        },
```

Comment:
This should be a bit more robust if they already have a prettier config:
```suggestion
        {
            name: () => `.prettierrc`,
            contentType: "json",
            content: ({ data }) => {
                data.useTabs ??= true;
                data.singleQuote ??= true;
                data.trailingComma ??= "none";
                data.printWidth ??= 100;
                data.plugins ??= [];
                data.overrides ??= [];

                const plugins: string[] = data.plugins;
                if (!plugins.includes("prettier-plugin-svelte")) {
                    data.plugins.unshift("prettier-plugin-svelte");
                }

                const overrides: { files: string | string[]; options?: { parser?: string } }[] = data.overrides;
                const override = overrides.find((o) => o?.options?.parser === "svelte");
                if (!override) {
                    overrides.push({ files: "*.svelte", options: { parser: "svelte" } });
                }
            },
```